agent_deploy/kubernetes/sysdig-agent-service.yaml

```diff
@@ -0,0 +1,13 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: sysdig-agent
+  labels:
+    app: sysdig-agent
+spec:
+  selector:
+     app: sysdig-agent
+  ports:
+  - protocol: TCP
+    port: 8765
+    targetPort: 7765
```

Comment:
Yeah we wanted the port for the service to be different than the port actually used within the container by the agent. We thought it would make potential debugging a bit easier.

Reply:
Good